### ISSUE TITLE
Phpunit 9.3.10 issue

### ISSUE THREAD

**Issue by coding-sunshine** [pestphp/pest#182]
Can't install with PHPUnit 9.3.10

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for pestphp/pest ^0.3.1 -> satisfiable by pestphp/pest[v0.3.1].
    - Conclusion: remove phpunit/phpunit 9.3.10
    - Conclusion: don't install phpunit/phpunit 9.3.10
    - pestphp/pest v0.3.1 requires phpunit/phpunit 9.3.7 || 9.3.8 -> satisfiable by phpunit/phpunit[9.3.7, 9.3.8].
    - Can only install one of: phpunit/phpunit[9.3.7, 9.3.10].
    - Can only install one of: phpunit/phpunit[9.3.8, 9.3.10].
    - Installation request for phpunit/phpunit (locked at 9.3.10, required as ^9.0) -> satisfiable by phpunit/phpunit[9.3.10].

```

**Comment by owenvoke** [comment]
Thanks for this, we're looking into this (see https://github.com/pestphp/website/pull/83). It's because of the use of internal classes.

**Comment by nunomaduro** [comment]
Fixed on version `v0.3.2`.
